Task_1/setup.py

```diff
@@ -28,7 +28,7 @@
     ],
     include_package_data=True,
     install_requires=[
-        'openfl @ git+https://github.com/securefederatedai/openfl.git@kta-intel/fets-2024-patch-1',
+        'openfl @ git+https://github.com/securefederatedai/openfl.git@84cff481b1ec8c92fc88c759ab5b8b6f950f9ca4',
```

Comment:
@kta-intel, is there any plan to incorporate your branch [[ref](https://github.com/securefederatedai/openfl/tree/kta-intel/fets-2024-patch-1)] into OpenFL develop?

It is still showing as unmerged as of today:

![image](https://github.com/user-attachments/assets/95728722-286d-49fe-b1a7-fbeb3728acac)

I think it makes sense for us to merge this PR as-is now (so that the rest of the code is up-to-date), but do a separate PR to point openfl to the latest tag (perhaps along with a simple unit test). Thoughts?